docs/wfc3tools/wf3ccd.rst

```diff
@@ -50,8 +67,10 @@ Parameters
         Name of the output FITS file.
 
     dqicorr : str, optional, default="PERFORM"
-        Update the dq array from bad pixel table. Allowed values are "PERFORM"
-        and "OMIT".
+        Update the dq array from bad pixel table, as well as flag the A-to-D saturation.
```

Comment:
i thought this step just initialized the DQ array, but im seeing in some places in the documentation that it flags saturation and/or sink pixels

Reply:
Indeed, DQICORR does more than just initialize pixels based on BPIXTAB.